### PR TITLE
Update DG for Review Mode

### PR DESCRIPTION
**Pull request by vuminhhieunus2019**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ diagrams.
 </div>
 
 **`Main`** has two classes
-called [`Main`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/Main.java)
-and [`MainApp`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java). It
+called [`Main`](https://github.com/AY2021S2-CS2103T-T13-3/tp/blob/master/src/main/java/seedu/address/Main.java)
+and [`MainApp`](https://github.com/AY2021S2-CS2103T-T13-3/tp/blob/master/src/main/java/seedu/address/MainApp.java). It
 is responsible for,
```
Comment (yeoutzer) [inline docs/DeveloperGuide.md:39]: Thank you for changing to the correct link. Same for the others.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -116,11 +118,9 @@ call.
 
 ![Structure of the Model Component](images/ModelClassDiagram.png)
 
-**
-API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
-
-The `Model`,
+**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
```
Comment (yeoutzer) [inline docs/DeveloperGuide.md:121]: Thank you for updating the diagram.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -165,15 +163,56 @@ It implements the following operations:
...
+##### Aspect: How user interacts in Review Mode.
+
+* **Alternative 1 (current choice):** interacts via `CommandBox` similar to the `MainWindow`
+    * Pros: Easy to implement.
+    * Cons: User experience might not be good since user needs to type many commands.
+
+* **Alternative 2:** interacts via keyboard
+    * Pros: Better user experience since user can move to next/previous flashcard a lot faster.
+    * Cons: Harder to implement.
+
```
Comment (yeoutzer) [inline docs/DeveloperGuide.md:211]: Good job on the documentation of implementing review feature.

**Review by yeoutzer — COMMENTED** [review]
Overall good job. Thank you for changing the diagram to fit our project. LGTM.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (marc-97) [inline docs/DeveloperGuide.md:121]: Should the link be from our repo instead?

**Review by marc-97 — COMMENTED** [review]
Looks good to me, thanks for updating the diagram

**Pull request merged**